scala/finch/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.7.3")
```

Comment:
can we use a less strict constraint ? I mean the promise here, is to ensure having a specific version of `finch`, we probably do not need to specify version of `sbt-native-packager` (we can use the latest, I think) ?


the think is to have the same build tool for each `scala` based frameworks

Reply:
Latest is now 1.8.0... we can possibly use the usual "range" thing I guess. can you make that change?

Reply:
my question was more to avoid pin this dependency
for example, by using http://ant.apache.org/ivy/history/latest-milestone/ivyfile/dependency.html#revision

Reply:
and I replied that I agree... we can set that plugin version something like `[1.7,2.0]`

Reply:
let me rephrase : is there a way to remove this version ? or tell **sbt** to use the last available (not depending on a range) ?

Reply:
by reading [here](https://ant.apache.org/ivy/history/2.3.0/ivyfile/dependency.html#revision) I think we can "try" with `latest.integration`

Reply:
seems 🆗

Reply:
but `sbt` warns about
```
[warn] There may be incompatibilities among your library dependencies; run 'evicted' to see detailed eviction warnings.
```